lib/gemfury/command/app.rb

```diff
@@ -198,23 +198,23 @@ def git_list
     end
   end
 
-  desc "git:rename", "Rename a Git repository"
+  desc "git:rename NAME", "Rename a Git repository"
   def git_rename(repo, new_name)
     with_checks_and_rescues do
       client.git_update(repo, :repo => { :name => new_name })
       shell.say "Renamed #{repo} repository to #{new_name}\n"
     end
   end
 
-  desc "git:reset", "Remove a Git repository"
+  desc "git:reset NAME", "Remove a Git repository"
```

Comment:
Would it be more clear to use `REPO_NAME`?

```suggestion
  desc "git:reset REPO_NAME", "Remove a Git repository"
```